imports/ui/pages/profile.jsx

```diff
@@ -87,13 +86,21 @@ class _Profile extends Component {
   }
 
   componentDidUpdate () {
-    if (!this.state.uploadActive) return
-    if (Meteor.isTest) return
-    new Dropzone('#profile-image-uploader', {
+    if (!this.state.uploadActive) {
+      return
+    }
+
+    if (Meteor.isTest) {
+      return
+    }
+
+    /* eslint-disable no-unused-vars */
+    // The drop
+    let dropzone = new Dropzone('#profile-image-uploader', {
       url: '/some/random/url',
       acceptedFiles: 'image/jpeg,image/png,image/gif',
       accept: (file, done) => {
-        let reader = new FileReader()
+        let reader = new window.FileReader()
```

Comment:
Again, just want to make sure this is the right thing to do.

Reply:
Same as my other comment